dev-system/compose.yaml

```diff
@@ -0,0 +1,41 @@
+services:
```

Comment:
When you say this docker compose can be 'potentially' used for local dev. Are you going to add how to do this in README?

Reply:
Sure I can add a bit of documentation